client/web/src/components/views/device-details-view.tsx

```diff
@@ -151,6 +151,23 @@ export default function DeviceDetailsView({
             </tbody>
           </table>
         </Card>
+        <Card noPadding className="-mx-5 p-5 details-card">
+          <h2 className="mb-2">Debug</h2>
+          <table>
+            <tbody>
+              <tr>
+                <td>TUN Mode</td>
+                <td>{node.TUNMode ? "true" : "false"}</td>
```

Comment:
```suggestion
                <td>{node.TUNMode ? "On" : "Off"}</td>
```

Reply:
Looks like admins console uses "Yes / No" so maybe that?

Reply:
That works!